pkg/clients/util.go

```diff
@@ -24,12 +24,12 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/sapcc/go-pagerduty"
+	"github.com/PagerDuty/go-pagerduty"
```

Comment:
github.com/sapcc/go-pagerduty

Reply:
nope - I kicked it out, because of changes in the object and there is no benefit of the fork anymore - the origin covers needed functionality

Reply:
https://github.com/sapcc/go-pagerduty/commit/304f7ea6b5b6d37787f9fa87bfaa6f4a417c4346 is still not upstream